lib/logstash/outputs/stomp.rb

```diff
@@ -57,11 +66,35 @@ def register
     
     connect
   end # def register
-  
-  def receive(event)
-      
 
-      @logger.debug(["stomp sending event", { :host => @host, :event => event }])
-      @client.send(event.sprintf(@destination), event.to_json)
+  public
+  def do_close
+    @logger.warn("Disconnecting from stomp broker")
+    @client.disconnect if @client.connected?
+  end # def do_close
```

Comment:
Oh yes, of course, you're right. I've amended this PR with that change.

I suggest to also mention that `close` method in the ["how to contribute" documentation](https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_output_plugin.html)